app/lunatech/lunchplanner/services/ReportService.scala

```diff
@@ -38,6 +38,21 @@ class ReportService @Inject()(
     attendees.map(menuAttendant => Report(menuAttendant.groupBy(_.date.toString).mapValues(_.map(_.name))))
   }
 
+  def getReportByLocationAndDate(month: Int): Future[ReportByDateAndLocation] = {
+    val baseDate = DateTime.now.withMonthOfYear(month)
+    val sDate = baseDate.withDayOfMonth(1)
+    val eDate = sDate.plusMonths(1).minusDays(1)
+
+    val attendees: Future[Seq[MenuPerDayReportByDateAndLocation]]  = menuPerDayService.getAllOrderedByDateFilterDateRange(new Date(sDate.getMillis), new Date(eDate.getMillis)).flatMap {
```

Comment:
very small thing: if you run`scalafmt` inside sbt to indent the code then this will look better :)